components/server/src/server.rs

```diff
@@ -410,7 +410,14 @@ where
             config.coprocessor.clone(),
         ));
 
-        let region_info_accessor = RegionInfoAccessor::new(coprocessor_host.as_mut().unwrap());
+        // Region stats manager collects region heartbeat for use by in-memory engine.
+        let enable_region_stats_manager =
```

Comment:
It's better to reference the config directly instead using a temp variable. Because once the config is changed, it cannot propagate automatically.  Ideally we should make config change take effect without restarting tikv.

Reply:
Or we need to have a callback when the config is updated, if using a temp variable.

Reply:
ah got it. see the comment. i'll take a look at how to see. the problem here would be passing a fairly unrelated piece of overall config here, but callback might work.

Reply:
done.